src/vm/assemblynative.hpp

```diff
@@ -112,6 +112,8 @@ class AssemblyNative
 
     static BOOL QCALLTYPE GetIsCollectible(QCall::AssemblyHandle pAssembly);
 
+    static INT32 QCALLTYPE ExecuteMainMethod(QCall::AssemblyHandle pAssembly, QCall::ObjectHandleOnStack args);
```

Comment:
Why not `#ifndef FEATURE_PAL` like for the other one? Because this one works everywhere?

Reply:
Yep.